### PR TITLE
Extending kickstarts for building centos images as well

### PR DESCRIPTION
**Pull request by ckannan**
Signed-off-by: Chandrasekar Kannan <kannan.chandrasekar@gmail.com>

### CODE REVIEW

File: kickstarts/Makefile [hunk 1]
```diff
@@ -3,8 +3,8 @@
 PRODUCT     := CloudRouter
 VERSION     := 1.0
 OS          := fedora
-OS_VERSION  := 20
-OS_TREE     := http://dl.fedoraproject.org/pub/fedora/linux/releases/20/Fedora/x86_64/os/
+OS_VERSION  := 21
```
Comment (abn) [inline kickstarts/Makefile]: Maybe move to 22?  And this should probably be a separate commit.

File: kickstarts/cloudrouter-centos-repo.ks [hunk 2]
```diff
@@ -0,0 +1,4 @@
+repo --name=centos --mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
```
Comment (abn) [inline kickstarts/cloudrouter-centos-repo.ks]: We need to look at how the *-repo.ks file usage has changed. Since cloudrouter-release packages are meant to handle this I suspect we can drop these. Additionally the $releasever variable will cause issues.

File: kickstarts/cloudrouter-centos-base.ks [hunk 3]
```diff
@@ -0,0 +1,34 @@
+%include cloudrouter-base.ks
+%include cloudrouter-centos-repo.ks
+
+%packages
+epel-release
+bird
```
Comment (abn) [inline kickstarts/cloudrouter-centos-base.ks]: should bird be part of the base image?

File: kickstarts/Makefile (hunk 1 above)
Comment (ckannan) [inline kickstarts/Makefile]: makes sense. I'll exclude this one.

File: kickstarts/cloudrouter-centos-repo.ks (hunk 2 above)
Comment (ckannan) [inline kickstarts/cloudrouter-centos-repo.ks]: I have booted the virt images that are created with this and have tried yum update and that runs successfully. But yes I agree with the suggestion that we should let cloudrouter-release packages handle these.

File: kickstarts/cloudrouter-centos-base.ks (hunk 3 above)
Comment (ckannan) [inline kickstarts/cloudrouter-centos-base.ks]: It needs to be excluded.

**Comment by ckannan** [comment]
Will submit another pull request incorporating some of the changes suggested here.
